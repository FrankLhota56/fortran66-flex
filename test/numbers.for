	I = 12
	I = 1 33
	X = 12.
	x = .5
	X = 1 000. + 0 . 3 3
	XD = 1 316 791 . 43 271 D + 04
	END