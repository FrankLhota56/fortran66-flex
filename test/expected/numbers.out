Line     1: ID(i)
Line     1: '='
Line     1: INT_LIT(12)
Line     2: EOS
Line     2: ID(i)
Line     2: '='
Line     2: INT_LIT(133)
Line     3: EOS
Line     3: ID(x)
Line     3: '='
Line     3: REAL_LIT(12.000000)
Line     4: EOS
Line     4: ID(x)
Line     4: '='
Line     4: REAL_LIT(0.500000)
Line     5: EOS
Line     5: ID(x)
Line     5: '='
Line     5: REAL_LIT(1000.000000)
Line     5: '+'
Line     5: REAL_LIT(0.330000)
Line     6: EOS
Line     6: ID(xd)
Line     6: '='
Line     6: DBL_PREC_LIT(13167914327.100000)
Line     7: EOS
Line     7: END
